external-crates/move/crates/move-compiler/src/unit_test/filter_test_members.rs

```diff
@@ -83,6 +83,7 @@ impl FilterContext for Context<'_> {
 
 const UNIT_TEST_MODULE_NAME: Symbol = symbol!("unit_test");
 const STDLIB_ADDRESS_NAME: Symbol = symbol!("std");
+pub const UNIT_TEST_POISON_FUN_NAME: Symbol = symbol!("unit_test_poison");
```

Comment:
Consider also using this where the function is inserted?

Reply:
I wish there was a face-palm emoji available here. The whole reason of introducing this constant in this file was to use it for function insertion...